types/test/test_main.c

```diff
@@ -16,4 +16,5 @@ TEST_MAIN(
     interpolator_immediate_change_test,
     test_ports,
     test_time_series_new,
-    test_time_after)
+    test_time_after,
+    test_time_delta)
```

Comment:
Were these tests just "lost" in the sense that they'd dropped out of this list?

Reply:
Yes, and because they weren't declared static nothing complained.